locales/en/messages.json

```diff
@@ -4784,6 +4784,10 @@
         "message": "<span class=\"message-negative\">Attention:</span> You need to configure and save FIRST the VTX Table at the bottom before you can make use of the $t(vtxSelectedMode.message) fields.",
         "description": "Message to show when the VTX is not supported in the VTX tab"
     },
+    "vtxMessageVerifyTable": {
+        "message": "<span class=\"message-negative\">Attention:</span> The values of the VTX Table have been loaded but not saved, you must verify and modify the values to be sure that that they are valid and legal at your country and then press the $t(vtxButtonSave.message) button.",
```

Comment:
I would probably go for this here:

The values of the VTX Table have been loaded, but not yet stored on the flight controller. You must verify and modify the values to be sure that that they are valid and legal in your country and then press the Save button to store them on the flight controller.

Reason: 'Save' has two meanings in this context.